node-definitions/arista/veos/vEOS.yaml

```diff
@@ -0,0 +1,64 @@
+id: vEOS
+general:
+  description: Arista vEOS
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - eth0
+      - eth2
+      - eth3
+      - eth4
+      - eth5
+      - eth6
+      - eth7
+      - eth9
+      - eth10
+      - eth11
+      - eth12
+      - eth13
+    serial_ports: 1
+    default_count: 13
+    loopback:
+      - l0
+ui:
+  description: |-
+    Arista vEOS
+
+    2 GB DRAM, 2 vCPU
+
+    [Link](https://www.arista.com/en/cg-veos-router/veos-router-overview)
+  visible: true
+  label_prefix: veos-
+  icon: switch
+  label: vEOS
+sim:
+  linux_native:
+    libvirt_domain_driver: kvm
+    driver: server
+    disk_driver: sata
+    ram: 2048
+    cpus: 2
+    cpu_limit: 100
+    nic_driver: virtio
+boot:
+  timeout: 1
+inherited:
+  image:
+    ram: true
+    cpus: true
+    cpu_limit: true
+    data_volume: true
+    boot_disk_size: true
+  node:
+    ram: true
+    cpus: true
+    cpu_limit: true
+    data_volume: true
+    boot_disk_size: true
+configuration:
+  generator:
+    driver: null
```

Comment:
pyATS does support EOS.  Can this device not do a day 0 or initial config?  It may not, but figured I'd check as that is a useful feature.

Reply:
Its a good question, I havne't tried. The initial boot of a fresh instance has ZTP enabled which needs to be disabled/cancelled before you can save the configuration. Im not sure if this would break that overall workflow but maybe something to consider in the fiuture?

Reply:
It's a good question, I haven't tried. The initial boot of a fresh instance has ZTP enabled which needs to be disabled/cancelled before you can save the configuration. Im not sure if this would break that overall workflow but maybe something to consider in the future?

Reply:
Okay.  Definitely worth playing with.  If you can't extract the config and boot from an initial config, that may be a problem for some.